lisk/config/alphanet/config.json

```diff
@@ -431,16 +431,16 @@
 					}
 				],
 				"defaultPassword": "elephant tree paris dragon chair galaxy"
-			},
-			"network": {
-				"list": [
-					{
-						"ip": "127.0.0.1",
-						"wsPort": 5000
-					}
-				],
-				"wsPort": 5000
 			}
+		},
+		"network": {
+			"list": [
+				{
+					"ip": "127.0.0.1",
+					"wsPort": 5000
+				}
+			],
+			"wsPort": 5000
```

Comment:
Please update the `update_config.js` to apply this change in the user configuration file. See this script for reference: 

https://github.com/LiskHQ/lisk-sdk/blob/2efbe9c6c4d3780e639d1850e09cb3b423def02e/lisk/scripts/update_config.js#L274-L281

Reply:
Created separate issue #3466